src/coreclr/src/jit/liveness.cpp

```diff
@@ -1623,7 +1623,11 @@ bool Compiler::fgComputeLifeTrackedLocalDef(VARSET_TP&           life,
 //    keepAliveVars - The current set of variables to keep alive regardless of their actual lifetime.
 //    varDsc        - The LclVar descriptor for the variable being used or defined.
 //    lclVarNode    - The node that corresponds to the local var def or use.
-void Compiler::fgComputeLifeUntrackedLocal(VARSET_TP&           life,
+//
+// Returns:
+//    `true` if the node is a dead store (i.e. all fields are dead); `false` otherwise.
```

Comment:
does it currently return true somewhere?

Reply:
Sorry - I meant to respond to this. Yes, it returns true under the `if (isDef)` condition:
```
            // None of the fields were live, so this is a dead store.
            if (!opts.MinOpts())
            {
                // keepAliveVars always stay alive
                VARSET_TP keepAliveFields(VarSetOps::Intersection(this, fieldSet, keepAliveVars));
                noway_assert(VarSetOps::IsEmpty(this, keepAliveFields));

                // Do not consider this store dead if the parent local variable is an address exposed local.
                return !varDsc.lvAddrExposed;
            }
```